frontend/app/src/pages/People/index.tsx

```diff
@@ -3,7 +3,7 @@ import { Route, Switch, useRouteMatch, BrowserRouter } from 'react-router-dom';
 import { PeoplePage } from './PeoplePage';
 import { PersonPage } from './PersonPage';
 
-export const People = (prop: any) => {
+export const People = (props: any) => {
```

Comment:
do we even need props here if we never use them?

Reply:
fixed in [d41f974](https://github.com/stakwork/sphinx-tribes/pull/533/commits/d41f9746c70d1077ddc372dfe179f490e628e1bd)